integration-tests/actions/vrfv2_actions/vrfv2_steps.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync"
 	"time"
 
+	"cosmossdk.io/errors"
```

Comment:
checked with Test Tooling team - we should use go's standard package for dealing with errors instead of _"cosmossdk.io/errors"_ package

`fmt.Errorf("%s: %w", "msg", err)`

e.g. [link](https://github.com/smartcontractkit/chainlink/blob/develop/integration-tests/actions/vrfv2_actions/vrfv2_steps.go#L69)